src/components/BorrowerProfile/CommentsAndWhySpecial.vue

```diff
@@ -287,6 +297,17 @@ export default {
 									id
 									authorName
 									authorImageUrl
+									authorLendingAction {
```

Comment:
When will this be available on prod?

Reply:
I can coordinate with merging this in. It is on dev now and should be in prod next release

Reply:
We'll just need to make sure we've deployed that to prod before we send out the UI release.